src/Exception/DeveloperDoesNotExistException.php

```diff
@@ -47,7 +47,7 @@ class DeveloperDoesNotExistException extends ApiException implements ApigeeEdgeE
    */
   public function __construct(string $email, string $message = 'Developer with @email email address not found. Running the developer sync could fix this.', int $code = 0, \Throwable $previous = NULL) {
     $this->email = $email;
-    $message = strtr($message, ['@email' => $email]);
+    $message = \Drupal::translation()->translate($message, ['@email' => $email]);
```

Comment:
https://www.drupal.org/docs/develop/coding-standards/php-exceptions

> All Exceptions should include an appropriate message and should not be translated. Only messages from the install and update system are currently translated as they are user facing.

This is not how you make an exception translated for users _on the UI_. If an exception can reach the front-end you should capture that with an exception subscriber and based on the exception you should render a translatable message there.

See https://github.com/apigee/apigee-edge-drupal/blob/8.x-1.x/src/EventSubscriber/EdgeExceptionSubscriber.php

Some related issues that I found with Google:
https://www.drupal.org/project/drupal/issues/2055851
https://www.drupal.org/project/search_api_solr/issues/3001220